pkg/nsx/client.go

```diff
@@ -176,13 +179,13 @@ func GetClient(cf *config.NSXOperatorConfig) *Client {
 	staticRouteClient := vpcs.NewStaticRoutesClient(restConnector(cluster))
 	natRulesClient := nat.NewNatRulesClient(restConnector(cluster))
 	vpcGroupClient := vpcs.NewGroupsClient(restConnector(cluster))
-	portClient := subnets.NewPortsClient(restConnector(cluster))
+	portClient := subnets.NewPortsClient(restConnectorAllowOverwrite(cluster))
```

Comment:
Do we confirm that NSX only exposes PI checking for subnetPort and ipAddressAllocation resource?

Reply:
Due to security reason, we only add the parameter when it's necessary.
we need to overwrite when handling the pre-created vpc.
so only subnetport/ipAddressAllocation needed.